src/promptflow/promptflow/azure/_restclient/flow_service_caller.py

```diff
@@ -134,7 +134,7 @@ def _get_user_identity_info(self):
         import jwt
 
         token = get_arm_token(credential=self._credential)
-        decoded_token = jwt.decode(token.token, options={"verify_signature": False})
+        decoded_token = jwt.decode(token, options={"verify_signature": False})
```

Comment:
consider adding a UT for this function `_get_user_identity_info`? so that we can avoid regression in the future on same case

Reply:
Do we have test for `pf.flows.test`? maybe we can add one if not

Reply:
add a UT for this func then?

Reply:
done, modified the flow.list test, which will cover this function.

Reply:
We will not add test for now, as adding the following two fields to the token will cause the existing test replay mode to fail.
![image](https://github.com/microsoft/promptflow/assets/23182548/5d68ee00-9bdd-4b4e-9218-c86e958bd804)

Reply:
![image](https://github.com/microsoft/promptflow/assets/23182548/2a8609de-9e67-4a06-89a1-54a288b1f672)

Reply:
shall we create task to track this?